packages/core/src/Stores/contract-store.js

```diff
@@ -433,6 +484,15 @@ function calculate_marker(contract_info, { accu_high_barrier, accu_low_barrier }
     if (tick_count >= 1) {
         if (!isDigitContract(contract_type)) {
             // TickContract
+            if (isResetContract(contract_type)) {
+                return {
+                    contract_info: toJS(contract_info),
+                    type: 'TickContract',
```

Comment:
type could be a constant of 'TickContract' and 'NonTickContract'

Reply:
Good idea, will do 👍🏻